tests/shared/src/test/scala/spire/math/RealScalaCheckSuite.scala

```diff
@@ -196,28 +201,6 @@ class RealScalaCheckSuite extends munit.ScalaCheckSuite {
     }
   }
 
-  // def sample1(name: String)(f: Real => Real): Unit = {
-  //   property(name) {
-  //     forAll { (x0: Rational, i0: Byte, j0: Byte) =>
-  //       val x = f(Real(x0.abs))
-  //       val i = (i0 & 0xff) % 250 + 1
-  //       val j = (j0 & 0xff) % 250 + 1
-  //       val (k1, k2) = if (i <= j) (i, j) else (j, i)
-  //       val v1 = x(k1)
-  //       val v2 = x(k2)
-  //       val v3 = Real.roundUp(Rational(v2, SafeLong(2).pow(k2 - k1)))
-  //       v1 == v3
-  //     }
-  //   }
-  // }
-
-  // sample1("sample1 id")(x => x)
-  // sample1("sample1 negate")(x => -x)
-  // sample1("sample1 +")(x => x + x)
-  // sample1("sample1 *")(x => x * x)
-  // sample1("sample1 sqrt")(_.sqrt())
-  // sample1("sample1 pow(2)")(_.pow(2))
```

Comment:
I wonder what this was. It's been commented for nearly a decade now it seems.

Reply:
Your guess is as good as mine. It seemed like a unused test that I couldn't really decipher.